b2sdk/http_constants.py

```diff
@@ -0,0 +1,22 @@
+######################################################################
+#
+# File: b2sdk/http_constants.py
+#
+# Copyright 2021 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
```

Comment:
please add a comment to explain this file is very useful to avoid circular imports